.github/workflows/test-wheel-linux.yml

```diff
@@ -221,6 +230,21 @@ jobs:
           pushd ./cuda_core
           pip install -r "tests/requirements-cu${TEST_CUDA_MAJOR}.txt"
           pytest -rxXs tests/
+
+          # It is a bit convoluted to run the Cython tests against CTK wheels,
+          # so let's just skip them. Also, currently our CI always installs the
+          # latest bindings (from either major version). This is not compatible
+          # with the test requirements.
+          if [[ "${{ inputs.local-ctk }}" == 1 && "${SKIP_CUDA_CORE_CYTHON_TEST}" == 0 ]]; then
+            pip install cython
```

Comment:
Every time I am stumbled by this bug when using Cython as a run-time dependency (outside of a build system): Since starting Python 3.12 distutils is removed, we need to add setuptools explicitly. Cython for some reason faced some pushback (https://github.com/cython/cython/issues/5751)...

Reply:
fixed in bc9595b